templates/etc_archivematica_archivematicaCommon_dbsettings.j2

```diff
@@ -2,4 +2,4 @@
 user=archivematica
 password=demo
 host=localhost
-db=MCP
+database=MCP
```

Comment:
I'll give it a shot. I was actually planning to do so but I read this:

http://dev.mysql.com/doc/refman/5.5/en/option-files.html
> The [client] option group is read by all client programs (but not by mysqld). This enables you to specify options that apply to all clients. For example, [client] is the perfect group to use to specify the password that you use to connect to the server. (But make sure that the option file is readable and writable only by yourself, so that other people cannot find out your password.) Be sure not to put an option in the [client] group unless it is recognized by all client programs that you use. Programs that do not understand the option quit after displaying an error message if you try to run them.

But it doesn't seem to break when `db` and `database` coexist.